lib/recurly/base.php

```diff
@@ -30,6 +30,21 @@ public static function _get($uri, $client = null)
   }
 
   /**
+   * Send a HEAD request to the URI, validate the response and return the headers.
+   * @param string Resource URI, if not fully qualified, the base URL will be appended
```

Comment:
Nit pick (I get that we're following the existing comment convention): should be "prepended" rather than "appended".